client/tests/integration_test.rs

```diff
@@ -833,7 +833,7 @@ mod make_transfer {
     const AMOUNT: &str = "30000000000";
     const TARGET_ACCOUNT: &str =
         "01aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
-    const TRANSFER_ID: &str = "1314";
+    const TRANSFER_ID: u64 = 1314;
```

Comment:
This means all clients' (if we have any :sweat_smile:) using Rust client in their bigger ecosystem will have to update? If I have scripts that use `casper-client` Rust binary, would they still be able to use a string version?

Reply:
Other clients using our Rust lib (either as a Rust lib or as a C lib) will need to update.  Usage of the binary is unaffected by this change - the binary currently errors out if a user passes something which doesn't parse as a `u64` for the transfer ID arg.